src/iguana/algorithms/Algorithm.h

```diff
@@ -7,6 +7,7 @@
 #include <variant>
 #include <vector>
 
+#include <fmt/ranges.h>
```

Comment:
we can probably remove this, since it's in `Logger.h` now
```suggestion
#include <fmt/ranges.h>
```